### PR TITLE
[PowerMock] LPS 151923

### PR DESCRIPTION
**Pull request by JanisZhang**
Hi @tinatian @dantewang,

This pr is to remove PowerMock (Misc) part two. Thanks.

https://issues.liferay.com/browse/LPS-151923

`modules/apps/.....archived`
`modules/apps/.....document-library-repository-cmis-impl`
`modules/apps/.....oauth2-provider`
`modules/apps/.....upload`
`modules/apps/.....journal`
`modules/apps/.....dynamic-data-mapping-form`
`modules/apps/.....data-engine-rest-impl`

### CODE REVIEW

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/test/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormDisplayContextTest.java [hunk 1]
```diff
@@ -874,40 +886,37 @@ private void _setUpLanguageUtil() {
...
 
 	private void _setUpPortalUtil() {
 		PortalUtil portalUtil = new PortalUtil();
 
-		portalUtil.setPortal(mock(Portal.class));
+		portalUtil.setPortal(Mockito.mock(Portal.class));
 
-		when(
+		Mockito.when(
 			PortalUtil.getHttpServletRequest(Matchers.any(RenderRequest.class))
```
Comment (dantewang) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/test/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormDisplayContextTest.java:907]: @JanisZhang Please SF this to 

```
Portal portal = Mockito.mock(Portal.class);

portalUtil.setPortal(portal);

Mockito.when(
    portal.getHttpServletRequest(......)
)...
```

This is more clear that you are mocking `Portal`, not mocking the static `PortalUtil` methods.

**Review by dantewang — CHANGES_REQUESTED** [review]

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/test/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormDisplayContextTest.java (hunk 1 above)
Comment (JanisZhang) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/test/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormDisplayContextTest.java]: @dantewang sure

**Comment by tinatian** [comment]
@shuyangzhou This pull is good to go.

Tina

**Review by dantewang — APPROVED** [review]

**Comment by shuyangzhou** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/116471

**Comment by shuyangzhou** [comment]
ci:close

**Pull request closed without merging**